pydis_site/apps/api/viewsets/bot/off_topic_channel_name.py

```diff
@@ -69,11 +68,11 @@ def get_object(self) -> OffTopicChannelName:
         name = self.kwargs[self.lookup_field]
         return get_object_or_404(queryset, name=name)
 
-    def get_queryset(self) -> QuerySet:
+    def get_queryset(self, **kwargs) -> QuerySet:
```

Comment:
I'm kind of 50 / 50 on overriding this to add our own arguments, because this
is also called by REST Framework from what I know. That said, we also have
other places where this method is called, and `.filter()` is called again on it
right after. Would it make more sense to stick with the definition as-is
(without the changes from `all`) and update the `get_queryset(**params)` call
from below to perform the `filter` itself?

Reply:
Right, I think its better to stick to the original definition, and we can perform a filter() instead of passing in params to queryset